content/nomos/updates/2024-03-11.md

```diff
@@ -0,0 +1,68 @@
+---
+title: 2024-03-11 Nomos weekly
+tags:
+  - nomos-updates
+date: 2024-03-11
+lastmod: 2024-03-11
+draft: false
+description: Weekly update of Nomos
+---
+## `network privacy and mixnet`
+
+### `Research`
+
+- Work in Progress (WIP): The [Mixnet Incentivization document](https://www.notion.so/Mixnet-Incentivization-6db9731a221b49878224afd916e2b3ac) has been initiated. Current open questions will be addressed, covering system design, mathematical analysis, and more.
+- Continuing work on the mixnet with staking, incorporating modifications based on feedback. A section has been added concerning the hidden bonus of deanonymization. It’s a straightforward observation that random assignment of adversarial nodes can, in some cases, lead to a higher number of adversarial paths than naively expected. Also, a section about discussing the latency and anonymity relationship has been started. WIP [document](https://www.notion.so/Mixnet-with-Staking-c8ec3bfd461f4989b3ebbcf4b4b15324#2e57424c79bd44b19c4c4dd624f544b2).
+- Analysis of the fraction of compromised paths in the mix network: using an asymptotic lower bound to estimate the probability that the fraction of compromised paths, α, belongs to the interval [α0​,α1​]. The analysis suggests that in the mixnet of size n=240 with L=3 layers sampled from N=800 nodes, where M=200 nodes are adversarial, α can be almost three times larger than the average (M/N)L which assumes a mixnet of infinite width. The parameters n=240, L=3, and N=800 are currently used in Nym’s mixnet. Here for ¼ of adversarial nodes, the fraction of compromised paths can be as high as 0.05. To compare, the average here is 0.02. Summary is provided [here](https://www.notion.so/Analysis-of-failures-in-the-mix-network-feeef349720842759c59785af71c7c59?pvs=4).
+
+### `development`
+
+- [All PRs](https://github.com/logos-co/nomos-node/pull/569) for Mixnet v1 implementation have been merged. We've taken some additional time to polish the code according to feedback.
+- One remaining PR we are working on is adding a compilation option to enable mixnet. We're going to always enable mixnet in production, but we've discussed that it's also good to remain in the libp2p-only compilation mode for development to unblock other dev topics until everything of mixnet becomes stable. Will be finished this early this week.
+
+## `testnet`
```

Comment:
Should go at the end, before Misc section.
Better to follow always the same order, for example as in the IFT document